openmmapi/src/TorchForce.cpp

```diff
@@ -104,3 +104,13 @@ void TorchForce::setGlobalParameterDefaultValue(int index, double defaultValue)
     ASSERT_VALID_INDEX(index, globalParameters);
     globalParameters[index].defaultValue = defaultValue;
 }
+
+void TorchForce::setProperty(const std::string& name, const std::string& value) {
+    properties[name] = value;
```

Comment:
We should throw an exception if the name doesn't match a supported property.  Otherwise there's a very high risk of errors.  If a user mistypes the name of a property, it will be silently ignored.